src/lib/api/builder.ts

```diff
@@ -670,6 +670,29 @@ export class BuilderAPI extends BaseAPI {
     }
   }
 
+  async publishCollection(collectionId: string, itemIds: string[], signedMessage: string, signature: string, qty: number, salt: string) {
```

Comment:
should this method be `publishTPCollection` ?

Reply:
agreed! renamed :)